### PR TITLE
Split plan alta and plan baja configuration files

### PR DESCRIPTION
**Pull request by Joseph-Developer**

### CODE REVIEW

File: configuration/pih/pih-config-mexico-plan-baja.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "welcomeMessage": "EMR de Compañeros en Salud - Plan de la Libertad",
+    "browserWarning": "Por favor utiliza <b>Google Chrome</b> como navagador",
+  
+    "site": "PLAN_DE_LA_LIBERTAD",
+  
+    "primaryIdentifierPrefix": "PLA",
+    "providerIdentifierPrefix": "PLA",
```
Comment (brandones) [inline configuration/pih/pih-config-mexico-plan-baja.json:8]:
```suggestion
    "site": "PLAN_DE_LA_LIBERTAD_BAJA",
  
    "primaryIdentifierPrefix": "PLB",
    "providerIdentifierPrefix": "PLB",
```

File: configuration/pih/pih-config-mexico-plan-alta.json [hunk 2]
```diff
@@ -2,15 +2,14 @@
   "welcomeMessage": "EMR de Compañeros en Salud - Plan de la Libertad",
   "browserWarning": "Por favor utiliza <b>Google Chrome</b> como navagador",
 
-  "site": "PLAN_DE_LA_LIBERTAD",
+  "site": "PLAN_DE_LA_LIBERTAD_ALTA",
 
-  "primaryIdentifierPrefix": "PLA",
-  "providerIdentifierPrefix": "PLA",
+  "primaryIdentifierPrefix": "PLAA",
+  "providerIdentifierPrefix": "PLAA",
```
Comment (brandones) [inline configuration/pih/pih-config-mexico-plan-alta.json:8]: No se si debemos cambiar esto.

@mogoodrich , are there consequences for changing the primary ID prefix of a server?

Comment (mogoodrich) [reply configuration/pih/pih-config-mexico-plan-alta.json:8]: For the primary identifier, it's not intended to be changed so it's not advisable... thinking it through, it *could* work, but we'd want to do some testing on a local/staging server first...  and it certainly wouldn't update the exisitng identifiers that had already been generated (though would be easy enough to write a liquibase changeset to do this).

@mseaton ?

Looking at the Provider Identifier, I think that can be changed, but, again, would be good to test.  (And we'd want to write a liquibase script to modify the existing identifiers, if that's what we want to do).

Comment (brandones) [reply configuration/pih/pih-config-mexico-plan-alta.json:8]: @Joseph-Developer , debemos dejar los prefixos y `site` para el servidor de plan alta como tal. Ok?

Comment (Joseph-Developer) [reply configuration/pih/pih-config-mexico-plan-alta.json:8]: ok, ahorita hago el cambio

Comment (Joseph-Developer) [reply configuration/pih/pih-config-mexico-plan-alta.json:8]: el nombre del .json si esta bien que lo haya cambiado ?

Comment (brandones) [reply configuration/pih/pih-config-mexico-plan-alta.json]: esta bien

File: configuration/pih/pih-config-mexico-plan-baja.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "welcomeMessage": "EMR de Compañeros en Salud - Plan de la Libertad",
+    "browserWarning": "Por favor utiliza <b>Google Chrome</b> como navagador",
+  
+    "site": "PLAN_DE_LA_LIBERTAD_BAJA",
+  
+    "primaryIdentifierPrefix": "PLAB",
+    "providerIdentifierPrefix": "PLAB",
```
Comment (brandones) [inline configuration/pih/pih-config-mexico-plan-baja.json:8]: No me gusta tanto que es el unico prefixo de 4 letras, pero no creo que sea una problema.

Comment (brandones) [reply configuration/pih/pih-config-mexico-plan-baja.json:8]: Yo chequé el logico de configuracion de generacion de IDs: https://github.com/PIH/openmrs-module-pihcore/blob/aecf029be61d6a6706c47902c17d3dd80f81b497/api/src/main/java/org/openmrs/module/pihcore/identifier/mexico/ConfigureMexicoIdGenerators.java#L37-L47

**Review by brandones — APPROVED** [review]

**Pull request merged**